book/peer_review.md

```diff
@@ -95,32 +110,36 @@ Be critical and clear, but not mean. Ask questions and set actions.
 
 Developers working in pairs can approve changes to code as it is written, however, key discussions from pair programming sessions should still be documented to demonstrate which aspects of the code have been reviewed and discussed.
 
 This blog post from the Government Digital Service provides [more detailed steps to apply pair programming](https://gds.blog.gov.uk/2018/02/06/how-to-pair-program-effectively-in-6-steps/).
 
-### Remote review
+### Separate review process
 
-Remote review involves sharing your code with a reviewer, and receiving a review in the form of constructive comments. We describe this as remote, in the sense that the code author does not need to be present for the review. This is often an iterative process, where the reviewer may make additional suggestions until they are satisfied with the code.
+A separate review involves sharing your code with a reviewer, and receiving written feedback. We describe this as separate, because code development and review are separate and the code author does not need to be present for the review. This type of review is an iterative process, where the reviewer may make additional suggestions until they are satisfied with the code changes.
 
 This form of review works best when changes to the code are small and frequent. Requesting review of small but regular changes reduces the burden on reviewers, relative to large review of a complete project. Similarly to pair programming, reviewing small changes to code allows you to catch issues sooner.
 
 ```{important}
 If a project is only reviewed when all of the code has been written, this significantly reduces the benefit of review.
 
-There is a much larger burden on the reviewer and any issues that are identified may take a lot of time to fix across the whole project. A reviewer might highlight that certain quality assurance practices have not been used - for example, there has not been enough documentation or automated testing in the project. It would take a substantial amount of effort to add documentation and testing for the whole project. While if this was identified earlier on, the improved practices would be easier to apply as the code is developed.
+There is a much larger burden on the reviewer and any issues that are identified may take a lot of time to fix across the whole project. A reviewer might highlight that certain quality assurance practices have not been used - for example, there has not been enough documentation or automated testing in the project. It would take a substantial amount of effort to add documentation and testing for the whole project. If this was identified earlier, the improved practices could be to apply as the remaining code is developed.
```

Comment:
```suggestion
There is a much larger burden on the reviewer and any issues that are identified may take a lot of time to fix across the whole project. A reviewer might highlight that certain quality assurance practices have not been used - for example, there has not been enough documentation or automated testing in the project. It would take a substantial amount of effort to add documentation and testing for the whole project. If this was identified earlier, the improved practices could be applied as the remaining code is developed.
```